pom.xml

```diff
@@ -54,5 +54,20 @@
 
     <properties>
         <openmrsPlatformVersion>2.6.0</openmrsPlatformVersion>
+		<webservices.restVersion>2.44.0</webservices.restVersion>
     </properties>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.10.1</version>
+				<configuration>
+					<source>1.8</source> <!-- Use 1.8 for Java 8, or change to 11, 17, etc. -->
+					<target>1.8</target> <!-- Use 1.8 for Java 8, or change to 11, 17, etc. -->
```

Comment:
You can also just use `8` for Java 8. IIRC, this has been the case since the rebrand of 1.6 as Java 6.